<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountManage</class>
 <widget class="QWidget" name="AccountManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover
{
background-color: rgb(180, 85, 100);
color:white;
}

QPushButton:hover:pressed
{
background-color:rgb(180, 85, 100);
color:white;
}

QPushButton
{
background-color:rgb(84, 115, 135);
color:white;
height: 30px;
font-size:12px;
width: 100px;
border-radius:2px;
spacing:2px;
padding: 0px 0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="6,4">
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QWidget" name="widgetControl" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <widget class="QPushButton" name="pushButtonSelect">
            <property name="geometry">
             <rect>
              <x>1266</x>
              <y>10</y>
              <width>80</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>查询报表</string>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/image/image/查询.png</normaloff>:/image/image/查询.png</iconset>
            </property>
           </widget>
           <widget class="QLabel" name="labelSup">
            <property name="geometry">
             <rect>
              <x>116</x>
              <y>12</y>
              <width>54</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string> 供应商：</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBoxState">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>30</y>
              <width>60</width>
              <height>22</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
           <widget class="QLabel" name="labelState">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>30</y>
              <width>66</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string> 单据状态：</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBoxSupplier">
            <property name="geometry">
             <rect>
              <x>174</x>
              <y>12</y>
              <width>60</width>
              <height>22</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
           <widget class="QLabel" name="labelType">
            <property name="geometry">
             <rect>
              <x>238</x>
              <y>12</y>
              <width>66</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string> 单据类型：</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBoxType">
            <property name="geometry">
             <rect>
              <x>308</x>
              <y>12</y>
              <width>60</width>
              <height>22</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
           <widget class="QLabel" name="labelDateType">
            <property name="geometry">
             <rect>
              <x>390</x>
              <y>12</y>
              <width>66</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string> 日期类型：</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBoxTimeType">
            <property name="geometry">
             <rect>
              <x>440</x>
              <y>12</y>
              <width>70</width>
              <height>22</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
           <widget class="QLabel" name="labelBeginTime">
            <property name="geometry">
             <rect>
              <x>510</x>
              <y>12</y>
              <width>66</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string> 开始时间：</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelEndTime">
            <property name="geometry">
             <rect>
              <x>690</x>
              <y>20</y>
              <width>66</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string> 结束时间：</string>
            </property>
           </widget>
           <widget class="QDateEdit" name="dateEditEndTime">
            <property name="geometry">
             <rect>
              <x>780</x>
              <y>20</y>
              <width>90</width>
              <height>22</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
           <widget class="QDateEdit" name="dateEditStartTime">
            <property name="geometry">
             <rect>
              <x>580</x>
              <y>12</y>
              <width>90</width>
              <height>22</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonExport">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>10</y>
              <width>80</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>导出</string>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/image/image/export.png</normaloff>:/image/image/export.png</iconset>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidgetAccount">
         <column>
          <property name="text">
           <string>供应商</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>单号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>操作时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>状态</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidgetAccount">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabAccountInfo">
        <attribute name="title">
         <string>订单详情</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel
{
background-color:rgb(180, 85, 100);
color:white;
font-size:14px;
font-weight:bold;
}</string>
           </property>
           <property name="text">
            <string> 基本信息</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="labelNumber">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>单号：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditNumberInfo">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLabel" name="label_20">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>状态：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditStateInfo">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="labelSupplier">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>供应商：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditSupplierInfo">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_19">
           <item>
            <widget class="QLabel" name="labelWareHouse">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>仓库：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditWareHouseInfo">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="labelDate">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>日期：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditDate">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QLabel" name="labelTime">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>时间：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditTime">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="labelRemark">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>备注：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditRemarkInfo">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButtonLog">
           <property name="text">
            <string> 处理记录</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelMaterial">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel
{
background-color:rgb(180, 85, 100);
color:white;
font-size:14px;
font-weight:bold;
}</string>
           </property>
           <property name="text">
            <string>原料</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidgetInfo">
           <column>
            <property name="text">
             <string>原料名称</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>条形码</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>单价</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>数量</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="labelTotalCount">
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelMoney">
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>5</width>
                  <height>5</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCommandLinkButton" name="commandLinkButtonSure">
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
